.environment/gitleaks/gitleaks-config.toml

```diff
@@ -12,6 +12,7 @@ title = "PRIME ReportStream Gitleaks Configuration"
         'yarn\.lock$',
     ]
     paths = [
+        '.environment/soap_service/',
```

Comment:
I am deliberately excluding all of the files under the `soap_service` folder because a lot of it looks like it could violate a gitleaks rules without being an actual leak.

Reply:
thank you on behalf of everyone in the future